generators/app/templates/src/contexts/tictactoe-shape.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I'm not sure this file is valid json-ld... From what I've seen, there's a few ways to do it, but Curie formatting of the keys and scoped contexts seem like the most common ways to do it.

Reply:
Further conversation needed.